src/sider/senoppfolging/KandidatVarsel.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { BodyShort, Box, Heading } from "@navikt/ds-react";
+import { tilLesbarDatoMedArUtenManedNavn } from "@/utils/datoUtils";
+
+const texts = {
+  ikkeSvartHeading: "Den sykmeldte har ikke svart",
+};
+
+interface Props {
+  varselAt: Date;
+}
+
+export function KandidatVarsel({ varselAt }: Props) {
```

Comment:
Kunne vi fått til et navn som sier noe mer om hva denne komponenten blir brukt til? 😊
Hva tenker du om noe sånn? 👇
```suggestion
export function KandidatIkkeSvart({ varselAt }: Props) {
```

Reply:
Ja, det er bedre, endrer :+1: